src/routes.js

```diff
@@ -93,4 +94,9 @@ export default [
     path: '/admin',
     component: Admin,
   },
+  {
+    name: 'Banner',
```

Comment:
I wonder whether something like "submit_banner" would be a slightly more appropriate name? It's not important though, the current one works just fine